move-mutator/src/lib.rs

```diff
@@ -58,26 +59,31 @@ pub fn run_move_mutator(
     info!(
         "Executed move-mutator with the following options: {options:?} \n config: {config:?} \n package path: {package_path:?}"
     );
+    // TODO: add some sanity checks for provided arguments - e.g. compute_coverage is useless when
+    // `move_sources` is used
```

Comment:
Those should be probably added in the CLI module and just called from here.

Reply:
Agreed - I'm gonna handle this in a separate PR anyway